platform/platform_windows.go

```diff
@@ -7,30 +7,95 @@ import (
 	"syscall"
 	"unsafe"
 
+	"golang.org/x/sys/windows"
 	"golang.org/x/sys/windows/registry"
 )
 
+var (
+	modNetapi32 = windows.NewLazyDLL("Netapi32.dll")
+	procNetWkstaGetInfo = modNetapi32.NewProc("NetWkstaGetInfo")
```

Comment:
question: are these `LazyProcs` (https://godoc.org/golang.org/x/sys/windows#LazyProc) safely reusable? From the docs I don't see any red flags, but want to make sure I ask.

Reply:
Yes.